addition of new language: Mongolia

bayartsogt-ya opened the pull request:
I am adding the new language templates for the hosted API.
Please review this for me. Thank you so much!

bayartsogt-ya committed:
addition of new language: Mongolia

I am adding the new language templates for the hosted API.
Please review this for me. Thank you so much!

julien-c commented on DefaultWidget.ts:
```diff
@@ -361,6 +361,48 @@ const MAPPING_BN: PerLanguageMapping = new Map([
+		`Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+	] ],
+	[ "summarization", [
+		`Mongolia (/mɒnˈɡoʊliə/ (About this soundlisten), Mongolian: Монгол Улс, transcription: Mongol Uls, Traditional Mongolian: ᠮᠤᠩᠭᠤᠯ ᠤᠯᠤᠰ, transliteration: Mongγol ulus) is a landlocked country in East Asia. Its area is roughly equivalent with the historical territory of Outer Mongolia, and that term is sometimes used to refer to the current state. It is sandwiched between Russia to the north and  ...
```
this is in EN, is this intended?

bayartsogt-ya replied:
@julien-c 

Sorry for the bad validation.
I just corrected the initialization (which was in English, now in Mongolian)

Thank you for your help!

julien-c reviewed (approved):
Thanks! Also cc'ing @mapmeld for info

bayartsogt-ya committed:
update mn summaraization

I just fixed the wrong initialization for `mn` summarization
it was fault that i copied `english` version of what I am doing

julien-c commented:
Deployed: see e.g. https://huggingface.co/bayartsogt/bert-base-mongolian-cased

bayartsogt-ya commented:
> Deployed: see e.g. https://huggingface.co/bayartsogt/bert-base-mongolian-cased

@julien-c 
Thank you so much
